samples/NativeModuleSample/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file shouldn't exist, `samples/NativeModuleSample/` is not itself a node project.